Allow skipping the media-size-supported check if
the required attribute isn't present in the printer.
This is the same behavior as ippserver.

--- a/tools/ippeveprinter.c
+++ b/tools/ippeveprinter.c
@@ -8720,6 +8720,7 @@ valid_job_attributes(
       else
       {
 	size = ippGetCollection(member, 0);
+	supported = ippFindAttribute(client->printer->attrs, "media-size-supported", IPP_TAG_BEGIN_COLLECTION);
 
 	if ((x_dim = ippFindAttribute(size, "x-dimension", IPP_TAG_INTEGER)) == NULL || ippGetCount(x_dim) != 1 ||
 	    (y_dim = ippFindAttribute(size, "y-dimension", IPP_TAG_INTEGER)) == NULL || ippGetCount(y_dim) != 1)
@@ -8727,11 +8728,10 @@ valid_job_attributes(
 	  respond_unsupported(client, attr);
 	  valid = 0;
 	}
-	else
+	else if (supported)
 	{
 	  x_value   = ippGetInteger(x_dim, 0);
 	  y_value   = ippGetInteger(y_dim, 0);
-	  supported = ippFindAttribute(client->printer->attrs, "media-size-supported", IPP_TAG_BEGIN_COLLECTION);
 	  count     = ippGetCount(supported);
 
 	  for (i = 0; i < count ; i ++)
